### PR TITLE
Feature firmwareanalysis fix

### PR DESCRIPTION
**Pull request by kalpeshbhatia000**
## Mandatory Checklist

- Please choose the target release of Azure PowerShell
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] N/A

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/FirmwareAnalysis/FirmwareAnalysis.Autorest/docs/Get-AzFirmwareAnalysisSummary.md [hunk 1]
```diff
@@ -66,19 +59,17 @@ SystemDataLastModifiedAt     :
 SystemDataLastModifiedBy     :
 SystemDataLastModifiedByType :
 Type                         : Microsoft.IoTFirmwareDefense/workspaces/firmwares/summaries
- }}
 ```
 
 {{ List all the analysis results summary for a firmware by analysis type CVE.
```
Comment (msJinLei) [inline src/FirmwareAnalysis/FirmwareAnalysis.Autorest/docs/Get-AzFirmwareAnalysisSummary.md:64]:
```suggestion
List all the analysis results summary for a firmware by analysis type CVE.
```

File: src/FirmwareAnalysis/FirmwareAnalysis.Autorest/docs/Get-AzFirmwareAnalysisSummary.md [hunk 2]
```diff
@@ -66,19 +59,17 @@ SystemDataLastModifiedAt     :
...
 }}
```
Comment (msJinLei) [inline src/FirmwareAnalysis/FirmwareAnalysis.Autorest/docs/Get-AzFirmwareAnalysisSummary.md:65]:
```suggestion
```

File: src/FirmwareAnalysis/FirmwareAnalysis.Autorest/examples/Get-AzFirmwareAnalysisSummary.md [hunk 3]
```diff
@@ -17,18 +16,16 @@ SystemDataLastModifiedAt     :
 SystemDataLastModifiedBy     :
 SystemDataLastModifiedByType :
 Type                         : Microsoft.IoTFirmwareDefense/workspaces/firmwares/summaries
- }}
 ```
 
 {{ List all the analysis results summary for a firmware by analysis type CVE. }}
```
Comment (msJinLei) [inline src/FirmwareAnalysis/FirmwareAnalysis.Autorest/examples/Get-AzFirmwareAnalysisSummary.md:21]:
```suggestion
List all the analysis results summary for a firmware by analysis type CVE.
```

**Review by msJinLei — COMMENTED** [review]
Please record test for `New-AzFirmwareAnalysisFirmwareFilesystemDownloadUrl`

File: src/FirmwareAnalysis/FirmwareAnalysis.Autorest/docs/Get-AzFirmwareAnalysisSummary.md (hunk 1 above)
Comment (kalpeshbhatia000) [reply src/FirmwareAnalysis/FirmwareAnalysis.Autorest/docs/Get-AzFirmwareAnalysisSummary.md:64]: Done

File: src/FirmwareAnalysis/FirmwareAnalysis.Autorest/docs/Get-AzFirmwareAnalysisSummary.md (hunk 2 above)
Comment (kalpeshbhatia000) [reply src/FirmwareAnalysis/FirmwareAnalysis.Autorest/docs/Get-AzFirmwareAnalysisSummary.md:65]: sure

File: src/FirmwareAnalysis/FirmwareAnalysis.Autorest/examples/Get-AzFirmwareAnalysisSummary.md (hunk 3 above)
Comment (kalpeshbhatia000) [reply src/FirmwareAnalysis/FirmwareAnalysis.Autorest/examples/Get-AzFirmwareAnalysisSummary.md]: done

**Comment by kalpeshbhatia000** [comment]
> Please record test for `New-AzFirmwareAnalysisFirmwareFilesystemDownloadUrl`

@msJinLei The backend is not ready for this endpoint yet. but it will be ready in next couple of weeks. So I did not provide any test case for it and skipped it. This was intentional.

**Review by msJinLei — APPROVED** [review]

**Pull request merged**